assets/src/js/postAuthorDescription.js

```diff
@@ -0,0 +1,23 @@
+export const setupPostAuthorDescription = () => {
```

Comment:
Maybe it's an option to use bootstrap for this?

Reply:
Like we do for the main navigation dropdown https://github.com/greenpeace/planet4-master-theme/blob/1511143660231f541c3de4d975c3627be74f4108/templates/navigation-bar.twig#L8-L12

Reply:
If I check [BS's docs on multiple targets](https://getbootstrap.com/docs/5.1/components/collapse/#multiple-targets), their example looks like it has similar markup but both elements hide/show at the same time. Not sure why in our case it does it differently.

It looks somewhat better if the ellipsis `span` is after the rest of the text. It still shows too long but it's hard to notice.

Reply:
In our case the ellipsis has the `show` class at the beginning, so the button toggles both, but depending on their class it either shows or hides. I'll test putting the ellipsis at the end, thanks!

Reply:
I've pushed a small change just setting `display: none;` for the `collapsing` class, I feel like it's a bit better...?

Reply:
It's still a bit weird because there's a delay when showing the rest of the text before it pops up. I suppose that delay is because BS would do a sliding transition, but in our case it's not sliding and it pops up at the end of the transition duration.

Reply:
I agree. I've pushed a new commit adding `transition: none;` and that seems to solve it!